samples/bluetooth/mesh/dfu/distributor/overlay-smp-bt-auth.conf

```diff
@@ -1,4 +1,4 @@
-CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN=y
+CONFIG_MCUMGR_TRANSPORT_BT_PERM_RW_AUTHEN=y
```

Comment:
Isn't this the default now if `CONFIG_BT_SMP` is set? If yes, then I think it can be removed from the config file.

Reply:
On second thought here: If `_AUTHEN` is the default, we might not need this overlay at all. Let me talk to my team about this.

Reply:
For some reason this comment has moved: It was supposed to apply to the `_PERM_RW_AUTHEN=y` line

Reply:
Sure, please be quick about it, as this PR has been open for quite some time and I was hoping to finalize it today (if CI passes). Apart from that, this PR consumes a lot of CI time and small project cleanups could be done in follow-up PRs by the project maintainers. My PR shouldn't change the behaviour of this MCUmgr Kconfig option.

Reply:
@PavelVPV at least this line is redundant then, right? Since we do enable the `CONFIG_BT_SMP` option in this same file which will select this by default?

Reply:
Yes, but lets keep it for clarity.